interfaces/automated-testing/rid/injection.yaml

```diff
@@ -0,0 +1,180 @@
+          Current version of the test.
+        schema:
+          type: string
+        in: query
+        required: true
+    delete:
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/DeleteTestResponse'
+          description: Requested test was deleted successfully.
+      operationId: deleteTest
+      summary: Delete test
+      description: Remove all injected test data associated with this test from the host Service Provider's system under test.
+      security:
+        - TestAuth:
+            - rid.inject_test_data
+components:
+  schemas:
+    CreateTestParameters:
+      description: 'A complete set of data to be injected into a Service Provider system under test.'
+      required:
+        - requested_flights
+      type: object
+      properties:
+        requested_flights:
+          description: >-
+            One or more logical flights, each containing test data to inject into the system.
+            Elements should be sorted in ascending order of `timestamp`.
+          type: array
+          items:
+            $ref: '#/components/schemas/TestFlight'
+    ChangeTestResponse:
+      description: ''
+      required:
+        - injected_flights
+        - version
+      type: object
+      properties:
+        injected_flights:
+          description: 'The complete set of test data actually injected into the Service Provider system under test.'
+          type: array
+          items:
+            $ref: '#/components/schemas/TestFlight'
+        version:
+          description: Version of test.  Used to delete test.
+          type: string
+          example: 1
+    DeleteTestResponse:
+      description: ''
+      required:
+        - injected_flights
+      type: object
+      properties:
+        injected_flights:
+          description: 'The complete set of test data deleted.'
+          type: array
+          items:
+            $ref: '#/components/schemas/TestFlight'
+    TestFlight:
+      description: 'The set of data to be injected into a Service Provider system under test for a single flight.'
+      required:
+        - injection_id
+        - telemetry
+        - details_responses
+      type: object
+      properties:
+        injection_id:
+          description: 'ID of the injected test flight.  Remains the same regardless of the flight ID/UTM ID reported in the system.'
+          type: string
+          example: 'edb7695f-8737-4b9f-91f8-e2afbb333f41'
+        telemetry:
+          description: >-
+            The set of telemetry data that should be injected into the system for this flight.
+            Each element follows the RIDAircraftState schema from the ASTM remote ID standard.
+          type: array
+          items:
+            $ref: 'https://raw.githubusercontent.com/uastech/standards/astm_rid_1.0/remoteid/canonical.yaml#/components/schemas/RIDAircraftState'
+        details_responses:
```

Comment:
`details_responses` is a bit difficult to understand, recommend to rename it to `test_flight_details_responses` or something like that.

Reply:
The object in which `details_responses` is defined is `TestFlight`; could your suggested field name be inferred by prefacing the field with the object type in which the field is located?  (`<TestFlight>.details_response`)

Reply:
Yes that would work, thanks!